packages/mwp-app-route-plugin/src/route.js

```diff
@@ -50,6 +50,9 @@ export default (languageRenderers: { [string]: LanguageRenderer }) => ({
 			'electrode-csrf-jwt': {
 				enabled: true, // need to generate tokens on page request
 			},
+			'mwp-tracking-plugin': {
+				click: () => true, // always track clicks on server render
```

Comment:
I can't find where/how this `click` function is used, where can I see that?

Reply:
The server plugin uses this `route.settings.plugins` value to determine whether or not the cookie should be consumed: https://github.com/meetup/meetup-web-platform/pull/554/files#diff-15492b88113fa366a51ac60655d9e077R14

I've also added docs in the README that should make it easier for future devs to follow